docs/source/DeveloperGuide/CodeDocumenting/index.rst

```diff
@@ -4,6 +4,8 @@
 Documenting your developments
 ==================================
 
+.. contents::
```

Comment:
Sorry but my searching skills are failing on this one. What is this `.. contents::` ?

Reply:
It creates a table of contents for the page

Reply:
that was my guess, but I could not find it in the doc. What is the difference with `toctree` ?

Reply:
like here: http://lcd-data.web.cern.ch/lcd-data/doc/ilcdiracdoc/DOC/Files/UserGuide/faq.html

Reply:
`toctree` creates the table of contents based on including a given list of files, `contents` does it for the local file (or subsection if configured, https://docutils.sourceforge.io/docs/ref/rst/directives.html#table-of-contents)
Because that file was becoming longer, I felt that it makes navigation easier.

Reply:
Ah now I find it in the doc page. Funny they did not actually write `.. contents::` anywhere...